tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -88,14 +101,17 @@ def _PartitionFn(
 
 
 @beam.ptransform_fn
-@beam.typehints.with_input_types(Union[tf.train.Example, bytes])
+@beam.typehints.with_input_types(Union[tf.train.Example, 
+                                       tf.train.SequenceExample, 
+                                       bytes])
 @beam.typehints.with_output_types(beam.pvalue.PDone)
 def _WriteSplit(example_split: beam.pvalue.PCollection,
                 output_split_path: Text) -> beam.pvalue.PDone:
   """Shuffles and writes output split as serialized records in TFRecord."""
 
   def _MaybeSerialize(x):
-    return x.SerializeToString() if isinstance(x, tf.train.Example) else x
+    return x.SerializeToString() if isinstance(x, (tf.train.Example,
+      tf.train.SequenceExample)) else x
```

Comment:
I feel this won't pass the lint

Reply:
yes, you are right, this is too long to fit in 2 lines with correct formatting